pkg/v1/builder/template/plugintemplates/Makefile.tmpl

```diff
@@ -39,7 +41,11 @@ go.mod go.sum: $(GO_SRCS)
 
 .PHONY: build-local
 build-local: ## Build the plugin
-	tanzu builder cli compile --version $(BUILD_VERSION) --ldflags "$(LD_FLAGS)" --path ./cmd/plugin --target local --artifacts artifacts/${GOHOSTOS}/${GOHOSTARCH}/cli
+	tanzu builder cli compile --version $(BUILD_VERSION) --ldflags "$(LD_FLAGS)" --path $(PLUGIN_PATH) --target local --artifacts artifacts/${GOHOSTOS}/${GOHOSTARCH}/cli
+
+.PHONY: build-plugin-local
+build-plugin-local: ## Build plugin's locally for given plugin name (match-expression) and path
+	tanzu builder cli compile --match "$(PLUGIN_NAME)" --version $(BUILD_VERSION) --ldflags "$(LD_FLAGS)" --path $(PLUGIN_PATH) --target local --artifacts artifacts/${GOHOSTOS}/${GOHOSTARCH}/cli
```

Comment:
Why do we want this target for the generated plugins?  It looks the same as `build-local` except for the `--match` flag.  However, I think there is usually only a single plugin in these generated repos, so using `--match` may not be useful.  But I'm just learning about this stuff so I can very well be wrong.

Reply:
After discussing with Anuj, we decided not to add new make targets in Makefile.tmpl